src/lib/stitching/gravity/stitching.ts

```diff
@@ -25,6 +25,10 @@ export const gravityStitchingEnvironment = (
         partner: Partner
       }
 
+      extend type ArtistSeries {
+        artists(page: Int, size: Int): [Artist]
```

Comment:
I would leave out `page` and `size` and just roll with the full list here.

Reply:
I tried removing, but then lost the ability to pass any args 🤔. I think I'll merge this as-is but let's follow up if you think this is still incorrect.

Reply:
odd! I was thinking to remove since we prefer connections for pagination. For cases where we just want the full list (and don't need connection/pagination), just to avoid a foot gun, removing args that aren't needed (specifically pagination ones), seemed proper.

Not sure that args need to be passed in at all for this field?

Reply:
Ah yes, so weird thing: the `artists` root field in the local schema is still a list, and not a connection, [even in v2](https://github.com/artsy/metaphysics/blob/3f0bdd6467d397405f0150483ae92c82f53928cb/_schemaV2.graphql#L7102-L7118). Not sure of the decision process behind that, but that's why this a has non-connection-y arg list.